pom.xml

```diff
@@ -44,12 +167,6 @@
         <dice-provider-emulator-version>1.1.1</dice-provider-emulator-version>
         <dice-provider-version>1.1.1</dice-provider-version>
         <x509-provider-version>1.1.6</x509-provider-version>
-
-        <!-- Properties to consolidate slf4j versions -->
```

Comment:
are these no longer used?

Reply:
Yep, the dependency management section now holds the "source of truth" on what version we are using.